staff/francesc-destruels/cv/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+</head>
+
+<body>
+    <header>
+        <h1>Curriculum Vitae</h1>
+    </header>
+    <nav>
+        <a href="#Basics">Basics</a>
+        <a href="#Formation">Formation</a>
+        <a href="#Profesional">Profesionals</a>
+        <a href="#Lenguages">Lenguages</a>
+        <a href="#About">About</a>
+    </nav>
+    <section>
+        <article>
+            <h3 id="Basics">Basics</h3>
+            <b>Full Name:</b> Francesc Destruels Rossello<br>
+            <b>Nationality:</b> Catalonia<br>
+            <b>Birth Country:</b> Catalonia<br>
+            <b>Birth Date:</b> 10/05/1989 <br>
+            <b>Ethnic:</b>Caucasian <br>
+            <b>Current Status:</b> Engaged<br>
+        </article>
+    </section>
+
+    <section>
+        <h3 id="Formation">Formation</h3>
+        <table>
+            <tr>
+                <td>
+                    <b>Titulation</b>
+                </td>
+                <td>
+                    <b>Year Completed</b> &#8195
+                </td>
+                <td>
+                    <b>Where</b>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    High School
+                </td>
+                <td>
+                    2007
+                </td>
+                <td>
+                    I.E.S. La Pineda, Badalona
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    S.T. Pathological Anatomy
+                </td>
+                <td>
+                    2009
+                </td>
+                <td>
+                    I.E.S LA Guineueta
+                </td>
+            </tr>
+        </table>
+    </section>
+    <section>
+
+        <h3 id="Profesional">Prefesional Experience</h3>
+        <table>
+            <tr>
+                <td>
+                    <b>Company</b>
+                </td>
+                <td>
+                    <b>Position</b> &#8195
+                </td>
+                <td>
+                    <b>Location</b> &#8195
+                </td>
+                <td>
+                    <b>Years (in-out)</b>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    Carrefour, S.A
+                </td>
+                <td>
+                    Cashier
+                </td>
+                <td>
+                    Badalona
+                </td>
+                <td>
+                    2008-09
```

Comment:
You can use [`<time>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) tag to  represent a specific period in time.